requirements.txt

```diff
@@ -1,10 +1,11 @@
-cryptography==2.3.1
+cryptography==2.8
 jsonschema==2.6.0
-lxml==4.2.1
+lxml==4.4.2
 ndg-httpsclient==0.5.0
 pyasn1==0.4.3
 pyOpenSSL==18.0.0
 PyYAML==3.13
 progressbar2==3.43.1
 intervaltree==3.0.2
 termcolor==1.1.0
+requests==2.22.0
```

Comment:
maybe bump others too? e.g. pyyaml looks pretty old

Reply:
I only upgraded what I had to in order to build on Windows without having to compile from source. I can look into upgrading some more, at least adding version ranges.

Reply:
We discussed this a bit offline and decided to prioritize this at a later date to unblock qa.